includes/data-stores/class-wc-customer-data-store.php

```diff
@@ -128,12 +136,14 @@ public function create( &$customer ) {
 	 * Method to read a customer object.
 	 *
 	 * @since 3.0.0
-	 * @param WC_Customer $customer
-	 * @throws Exception
+	 * @param WC_Customer $customer Customer object.
+	 * @throws Exception If invalid customer.
 	 */
 	public function read( &$customer ) {
+		$user_object = get_user_by( 'id', $customer->get_id() );
```

Comment:
Put the customer ID check here so get_user_by is not called for no reason. ternary operator.

Reply:
Done in commit 383160a.